packages/cli/commands/init.js

```diff
@@ -126,27 +130,44 @@ exports.handler = async options => {
     process.exit(EXIT_CODES.ERROR);
   }
 
+  const doesOtherConfigFileExist = configFileExists(!useHiddenConfig);
+  if (doesOtherConfigFileExist) {
+    const path =
+      (c && path.join(getCwd(), c)) || getConfigPath('', !useHiddenConfig);
```

Comment:
I think we may want to omit the `(c && path.join(getCwd(), c))` portion of this. If this IF block triggers, it means that  `configFileExists()` returned true. That util doesn't account for the `--c` flag. So ideally we'd be showing the user the exact config file that triggered the error, which (I think) would be the one returned by `getConfigPath()`.

This also got me wondering, should we allow devs to use the `--c` and `--useHiddenConfig` flags simultaneously?

Reply:
I've omitted `(c && path.join(getCwd(), c))`. 

We do include a check for whether the c and useHiddenConfig flags both exist:

https://github.com/HubSpot/hubspot-cli/blob/2f261c2a8b0baa6e00e1537ccd4dbb76ddc43758/packages/cli/commands/init.js#L140-L143

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1177/commits/44cb0a0a0b10961680f9baf9372ab0b04ab65389